nautobot_secrets_providers/providers/aws.py

```diff
@@ -99,3 +99,63 @@ def get_value_for_secret(cls, secret, obj=None, **kwargs):
         except KeyError as err:
             msg = f"The secret value could not be retrieved using key {err}"
             raise exceptions.SecretValueNotFoundError(secret, cls, msg) from err
+
+
+class AWSSystemsManagerParameterStore(SecretsProvider):
+    """A secrets provider for AWS Parameter Store."""
+
+    # Documentation: https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html
```

Comment:
This could be part of the docstring instead of a comment, for improved discoverability.